src/ray/common/id.cc

```diff
@@ -129,6 +129,21 @@ const TaskID GenerateTaskId(const DriverID &driver_id, const TaskID &parent_task
   return TaskID::FromBinary(std::string(buff, buff + TaskID::Size()));
 }
 
+const ActorHandleID ComputeNextActorHandleId(const ActorHandleID &actor_handle_id,
```

Comment:
How about putting this method to a separated file `id_util.h` to keep `id.h` clean.

Reply:
I just followed `GenerateTaskId`. Maybe move it later?